packages/chaire-lib-common/src/utils/osm/OsmOverpassDownloader.ts

```diff
@@ -119,6 +141,38 @@ class OsmOverpassDownloaderImpl implements OsmOverpassDownloader {
         return osmToGeojson(jsonContent) as GeoJSON.FeatureCollection;
     }
 
+    /**
+     * Download data from openstreetmap API, convert it to geojson and write it to
+     * a geojson file
+     * If the file exist, it will be overwritten
+     *
+     * See downloadGeojson for params
+     */
+    public async writeGeojson(
+        filename: string,
+        boundPoly: GeoJSON.Polygon | GeoJSON.FeatureCollection | GeoJSON.Feature,
+        overpassXmlQueryString: string = allWaysAndRelationsXmlQuery
+    ): Promise<boolean> {
+        const response = await this.downloadData(boundPoly, overpassXmlQueryString, 'json');
+        const streamPipeline = promisify(pipeline);
+
+        console.log('Writing osm geojson data to ' + filename);
+
+        // Inspired by a conversion with ChatGPT :)
+        const writeStream = fs.createWriteStream(filename);
+        const jsonStream = JSONStream.parse('elements.*');
+        jsonStream.on('data', (element) => {
```

Comment:
How and why is this write method different from the other write methods, who just call this.write. Maybe comment why you need a separate implementation here.

Reply:
If I copy the desciption from the commit message, is it enough?

Reply:
yes, so that's what osmToGeojson was for...